web/tests/compose_validate.test.js

```diff
@@ -77,16 +84,17 @@ function stub_message_row($textarea) {
 
 test_ui("validate_stream_message_address_info", ({mock_template}) => {
     mock_banners();
-    const sub = {
+
+    const party_sub = {
```

Comment:
"social" subs were being added multiple times in this test file, so I just named this one clearly differently to be less confusing